automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCEDataSyncingSpec.scala

```diff
@@ -21,6 +22,12 @@ import scala.concurrent.duration._
 class NotebookGCEDataSyncingSpec extends RuntimeFixtureSpec with NotebookTestUtils {
   override def enableWelder: Boolean = true
 
+  override def withFixture(test: NoArgTest) =
```

Comment:
Maybe this could be moved to `RuntimeFixtureSpec` to avoid duplication (not all tests extend `RuntimeFixtureSpec`, but many do)